Signal/src/ViewControllers/ConversationView/Cells/OWSContactOffersCell.m

```diff
@@ -15,7 +15,7 @@
 
 @interface OWSContactOffersCell ()
 
-@property (nonatomic, nullable) OWSContactOffersInteraction *interaction;
+@property (nonatomic, nullable, readonly) OWSContactOffersInteraction *interaction;
```

Comment:
Do we even need this private property any more?  Isn't the accessor below sufficient?

Reply:
Ok - removed.